routes/newProductSubmission.js

```diff
@@ -0,0 +1,85 @@
+ var mysql = require("mysql");
+ var config = require("konfig")();
+ var express = require("express");
+ var router = express.Router();
+ /* Is there a way to do this without the relative path? */
+ var db = require("../imp_modules/impdb.js");
+
+ /*
+ Usage:
+ localhost:50001/ProductID/Name/Customer/Description/DateCreated
```

Comment:
usage is actually: 
localhost:50001/newProductSubmission/ProductID/Name/Customer/Description/DateCreated

I would prefer if we could set a standard for what variables vs static stuff in a route looks like:
localhost:50001/newProductSubmission/:ProductID/:Name/:Customer/:Description/:DateCreated
or 
localhost:50001/newProductSubmission/{ProductID}/{Name}/{Customer}/{Description}/{DateCreated}

or something similar. We should also be defining what each variable means in this comment. For examples take a look at:

http://apidocjs.com/